package.json

```
@@ -69,7 +69,7 @@
     "typescript": "^4.0.2"
   },
   "engines": {
-    "node": "^12.20.0 || ^14.14.0 || >=16.0.0"
+    "node": "^14.14.0 || ^16.0.0 || >=18.0.0"
```

I would like to increase this to `14.18` so that we can use `node:` protocol imports and enable [unicorn/prefer-node-protocol](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-node-protocol.md).

https://nodejs.org/api/esm.html#node-imports

```suggestion
    "node": "^14.18.0 || ^16.0.0 || >=18.0.0"
```